undertow/src/main/java/org/wildfly/extension/undertow/logging/UndertowLogger.java

```diff
@@ -326,4 +327,8 @@
 
     @Message(id = 76, value = "Cannot remove resource of type %s")
     IllegalArgumentException cannotRemoveResourceOfType(String type);
+
+    @LogMessage(level = DEBUG)
+    @Message(id = 77, value = "HttpHandler for mod_cluster MCMP created.")
```

Comment:
no need to add messages below INFO to this file, as we have no need to localize debug messages.